### PR TITLE
Added aaryachopkar.json

### PR DESCRIPTION
**Pull request by accodes21**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot (260)

## Note to reviewers

Thank you in advance!

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1549"></a>

### CODE REVIEW

**Review by Kartik2220 — APPROVED** [review]
code looks good

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: public/data/aaryachopkar.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        "url": "https://github.com/accodes21",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/ChopkarAarya",
+        "icon": "twitter"
+      },
+      {
+        "name": "Connect with me on LikedIN",
```
Comment (kumarsonsoff3) [inline public/data/aaryachopkar.json:18]:
```suggestion
        "name": "Connect with me on LinkedIn",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
- Update your filename to your `username.json`, which should be `accodes21.json`
- Correct the small typo below! 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Comment by accodes21** [comment]
All mentioned Changed were made.
Thank you!

File: public/data/accodes21.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        "url": "https://github.com/accodes21",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/ChopkarAarya",
+        "icon": "twitter"
+      },
+      {
+        "name": "Connect with me on LikedIn",
```
Comment (kumarsonsoff3) [inline public/data/accodes21.json:18]: Same typo again! 

```suggestion
        "name": "Connect with me on LinkedIn",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant

**Comment by accodes21** [comment]
Sorry for the typo
Thanks for your patience and help!

**Review by kumarsonsoff3 — APPROVED** [review]
Good to goo! 🚀

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant, Maintainable

**Pull request merged**